third_party/phantomjs/BUILD

```diff
@@ -11,7 +11,7 @@ genrule(
     cmd = " && ".join([
         "IN=$$(pwd)/$(SRCS)",
         "OUT=$$(pwd)/$@",
-        "TMP=$$(mktemp -d)",
+        "TMP=$$(mktemp -d tmp.XXXXXXXXXX)",
```

Comment:
Out of curiosity, I've seen some bazel code that does it this way: `TMP=$$(mktemp -d $${TMPDIR:-/tmp}/phantomjs.XXXXXXXX)`. Do you know if there's any real difference between the two approaches?